packages/calypso-e2e/src/media-helper.ts

```diff
@@ -66,7 +66,8 @@ export function createTestFile( {
 	sourceFileName: string;
 	testFileName?: string;
 } ): string {
-	let fileName = getTimestamp();
+	let fileName = `${ getTimestamp() }${ getRandomInteger( 100, 999 ) }`;
+	console.log( fileName );
```

Comment:
Did you want to keep the log in here? Or was this just for debugging? Either is fine, just wanted to double check :)